test/test_jit.py

```diff
@@ -1564,6 +1564,187 @@ def forward(self, x):
                    .check('GetAttr[name="_quantized_weight"]') \
                    .run(m._c._get_method('forward').graph)
 
+    def test_fold_quantize_freeze(self):
```

Comment:
Is there a test of save/load of the freezed model? Something like:
```
ref = m.forward(input)
mf = torch._C._freeze_module(m._c)
mf.save(buf) # save frozen model
mf_loaded = load(buf) # load frozen model
output = mf_loaded.forward(input) # forward on loaded model
assertEQ(ref, output)
```

Reply:
Added